### PR TITLE
Add GRvDV 2023 form factor constraints

### PR DESCRIPTION
**Pull request by mreboud**

### CODE REVIEW

File: eos/constraints/B-to-P-form-factors.yaml [hunk 1]
```diff
@@ -736,6 +736,31 @@ B->K::FormFactors[f_+,f_T]@KR:2017A:
...
+B->K::FormFactors[parametric,BFW]@GRvDV:2023A:
+    type: MultivariateGaussian(Covariance)
+    observables:
+      [ 'B->K::a^f+_0@BFW2010', 'B->K::a^f+_1@BFW2010', 'B->K::a^f+_2@BFW2010',
+                                'B->K::a^f0_1@BFW2010', 'B->K::a^f0_2@BFW2010',
+        'B->K::a^fT_0@BFW2010', 'B->K::a^fT_1@BFW2010', 'B->K::a^fT_2@BFW2010' ]
+    kinematics:
+      [ {}, {}, {}, {}, {}, {}, {}, {} ]
+    options:
+      [ {'B->K::tp@BFW2010': 30.261001}, {'B->K::tp@BFW2010': 30.261001}, {'B->K::tp@BFW2010': 30.261001},
```
Comment (dvandyk) [inline eos/constraints/B-to-P-form-factors.yaml:748]: I think this is correct. However, the Github syntax highlighting is playing a prank on me. @mreboud can you confirm that the syntax is correct here? See the following:
image

**Review by dvandyk — APPROVED** [review]
Generally approved, but please see the one comment.

File: eos/constraints/B-to-P-form-factors.yaml (hunk 1 above)
Comment (mreboud) [reply eos/constraints/B-to-P-form-factors.yaml:748]: This is correct, gh is lost

Comment (mreboud) [reply eos/constraints/B-to-P-form-factors.yaml:748]: this is due to the `:`

Comment (dvandyk) [reply eos/constraints/B-to-P-form-factors.yaml:748]: Very good. Will merge!

**Pull request merged**